fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -452,6 +452,8 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
     public static final String SCAN_USE_QUERY_MEM_RATIO = "scan_use_query_mem_ratio";
     public static final String CONNECTOR_SCAN_USE_QUERY_MEM_RATIO = "connector_scan_use_query_mem_ratio";
     public static final String CONNECTOR_SINK_COMPRESSION_CODEC = "connector_sink_compression_codec";
+
+    public static final String CONNECTOR_SINK_TARGET_MAX_FILE_SIZE = "connector_sink_target_max_file_size";
```

Comment:
```suggestion
    public static final String CONNECTOR_SINK_TARGET_MAX_FILE_SIZE = "connector_sink_target_file_size";
```

Reply:
I do think target_max_file_size is more accessible. 🤔

Reply:
That's is Okay. But IMO they mean the same thing even if max is dropped.
Max means the upper bound, target is the soft upper bound.
If you keep both of two, a little confusing.